pkg/synthetictests/operators.go

```diff
@@ -28,6 +29,33 @@ func testUpgradeOperatorStateTransitions(events monitorapi.Intervals) []*junitap
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/OAuthServerRouteEndpointAccessibleController_EndpointUnavailable[ :].*"):   "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/OAuthServerServiceEndpointAccessibleControllerAvailable[ :].*"):            "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/ReadyIngressNodesAvailable[ :].*"):                                         "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/authentication condition/Available status/False reason/WellKnownAvailable[ :].*"):                                                 "https://bugzilla.redhat.com/show_bug.cgi?id=2010539",
+		regexp.MustCompile(".*clusteroperator/console condition/Available status/False reason/RouteHealthAvailable[ :].*"):                                                      "https://bugzilla.redhat.com/show_bug.cgi?id=2094517",
+		regexp.MustCompile(".*clusteroperator/console condition/Available status/False reason/RouteHealth_FailedGet[ :].*"):                                                     "https://bugzilla.redhat.com/show_bug.cgi?id=2094517",
+		regexp.MustCompile(".*clusteroperator/console condition/Available status/False reason/RouteHealth_RouteNotAdmitted[ :].*"):                                              "https://bugzilla.redhat.com/show_bug.cgi?id=2094517",
+		regexp.MustCompile(".*clusteroperator/dns condition/Available status/False reason/DNSUnavailable[ :].*"):                                                                "https://bugzilla.redhat.com/show_bug.cgi?id=1939723",
+		regexp.MustCompile(".*clusteroperator/image-registry condition/Available status/False reason/NoReplicasAvailable[ :].*"):                                                "https://bugzilla.redhat.com/show_bug.cgi?id=2081552",
+		regexp.MustCompile(".*clusteroperator/kube-storage-version-migrator condition/Available status/False reason/KubeStorageVersionMigratorAvailable[ :].*"):                 "https://bugzilla.redhat.com/show_bug.cgi?id=1948087",
+		regexp.MustCompile(".*clusteroperator/kube-storage-version-migrator condition/Available status/False reason/KubeStorageVersionMigrator_Deploying[ :].*"):                "https://bugzilla.redhat.com/show_bug.cgi?id=1948087",
+		regexp.MustCompile(".*clusteroperator/machine-config condition/Available status/False reason/Cluster not available[ :].*"):                                              "https://bugzilla.redhat.com/show_bug.cgi?id=2051075",
+		regexp.MustCompile(".*clusteroperator/storage condition/Available status/False reason/AWSEBSCSIDriverOperatorCR_AWSEBSDriverControllerServiceController_Deploying .*"):  "https://bugzilla.redhat.com/show_bug.cgi?id=2081557",
```

Comment:
I'd suggest `reason/.*CSIDriverOperatorCR_.*DriverControllerServiceController_Deploying` to catch all CSI driver operators. oVirt, IBM cloud, Alibaba, vSphere and OpenStack will have the same blips during upgrade.

Reply:
and is this the right Bugzilla tracker for those blips?

Reply:
Yes. I will flip it back to Assigned once the current fixes merge.